docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -381,19 +427,23 @@ Result:
 
 ## CAST(x, T) {#type_conversion_function-cast}
 
-Converts ‘x’ to the ‘t’ data type. The syntax CAST(x AS t) is also supported.
+Converts input value `x` to the `T` data type. The syntax `CAST(x AS t)` is also supported.
 
```

Comment:
Note, that if value `x` does not fit the bounds of type `T`, the function will overflow. For example, `CAST(-1, 'UInt8')` will return `255`.